addons/mail/static/src/composer/composer.xml

```diff
@@ -98,6 +99,9 @@
                 <div t-if="thread and thread.type !== 'chatter' and !compact">
                     <Typing channel="thread" size="'medium'"/>
                 </div>
+                <div t-if="state.gifKeyboardOpen" class="o-mail-Composer-gif-picker">
+                    <GifPicker onSelected.bind="sendGifMessage" close.bind="closeGifKeyboard" />
```

Comment:
`<GifPicker t-if="state.gifKeyboardOpen" onSelected.bind="sendGifMessage" close.bind="closeGifKeyboard" />` and make classes as props for the `GifPicker`